lms/djangoapps/discussion/rest_api/forms.py

```diff
@@ -186,3 +187,15 @@ def clean_rolename(self):
 
             self.cleaned_data['role'] = role
             return rolename
+
+
+class TopicListGetForm(Form):
+    """
+    Form for the topics API get query parameters.
+    """
+    topic_id = CharField(required=False)
+    order_by = ChoiceField(choices=TopicOrdering.choices, required=False)
+
+    def clean_topic_id(self):
+        topic_ids = self.cleaned_data.get("topic_id", None)
+        return set(topic_ids.strip(',').split(',')) if topic_ids else None
```

Comment:
What will be the value of topic_ids in the queryparam? I suspect its `topic_id=topic_id_1,topid_id_2`. My question is why you are splitting the value twice here?

Reply:
The value of topic ids will be as you mentioned "topic_id_1,topid_id_2" etc. 

The value is not being split twice here, the first function is `strip` which will remove excess commas, so that if you have a dangling comma it will be removed. Then we do a split to get the list of topics.

Reply:
Sorry about that, my bad. I miss interpreted `strip` as `split`. :(